collections/distinct.ts

```diff
@@ -6,6 +6,8 @@
  * Example:
  *
  * ```typescript
```

Comment:
Can we do a quick normalisation of the language attributes as-well? `s/typescript/ts/g`